jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -1160,6 +1160,14 @@ properties:
                 title: Filter Offset reference file name
                 type: string
                 fits_keyword: R_FILOFF
+          firstframe:
```

Comment:
Because this step doesn't use a reference file we don't need the new entries in the schema to define a R_FRSTFR keyword. We only need the S_FRSTFR keyword for the completion status. So the new lines 1163-1170 can be removed.

Reply:
It is correct that this step does not use a reference file as it is currently written.  But it will need one when we get a true correction algorithm and not just flagging.  Should I still delete these lines?

Reply:
Need @jemorrison to answer that.

Reply:
There is a desire one day to have a real correction for the first frame. This may happen after build 7.2 (certainly not for 7.2). I do not know of anyone on MIRI team activity working on a correction algorithm - it is just a hope.  So this would probably fall in the "place holder" status and I know you guys don't really want "place holders" in the pipeline. So I think at this time I would remove the reference to the first frame reference files.